### PR TITLE
fix: proxy test cases

### PR DESCRIPTION
**Pull request by mihir-4116**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: src/v0/destinations/azure_event_hub/transform.js [hunk 1]
```diff
@@ -5,20 +5,6 @@ function process(event) {
   const { message, destination } = event;
   const integrationsObj = getIntegrationsObj(message, "azure_event_hub");
   const topic = integrationsObj?.topic || destination.Config?.topic;
-  // TODO: Remove commented lines after server release
```
Comment (saikumarrs) [inline src/v0/destinations/azure_event_hub/transform.js:8]: Please revert this change but refactor the code.

File: src/v0/destinations/kafka/transform.js [hunk 2]
```diff
@@ -76,21 +76,6 @@ const process = event => {
     filterConfigTopics(message, destination) ||
     destination.Config?.topic;
 
-  // TODO: uncomment this when v.1.3.0 of server is avialble in all envs
```
Comment (saikumarrs) [inline src/v0/destinations/kafka/transform.js:79]: Revert this and refactor it. CC @utsabc

File: src/v0/destinations/salesforce/transform.js [hunk 3]
```diff
@@ -363,9 +360,7 @@ const processRouterDest = async inputs => {
       400,
       `Authorisation failed: ${error.message}`,
       {
-        destType: "SALESFORCE",
-        stage: TRANSFORMER_METRIC.TRANSFORMER_STAGE.TRANSFORM,
-        scope: TRANSFORMER_METRIC.MEASUREMENT_TYPE.AUTHENTICATION.SCOPE
+        destType: "SALESFORCE"
```
Comment (saikumarrs) [inline src/v0/destinations/salesforce/transform.js:363]: Even this is not required

**Review by saikumarrs — CHANGES_REQUESTED** [review]

File: src/v0/destinations/kafka/transform.js (hunk 2 above)
Comment (mihir-4116) [reply src/v0/destinations/kafka/transform.js:79]: @utsabc do i need to refactor and uncomment it or keep it commented??

File: src/v0/destinations/azure_event_hub/transform.js (hunk 1 above)
Comment (mihir-4116) [reply src/v0/destinations/azure_event_hub/transform.js:8]: done

File: src/v0/destinations/kafka/transform.js (hunk 2 above)
Comment (mihir-4116) [reply src/v0/destinations/kafka/transform.js:79]: done

File: src/v0/destinations/salesforce/transform.js (hunk 3 above)
Comment (mihir-4116) [reply src/v0/destinations/salesforce/transform.js]: done

**Review by utsabc — APPROVED** [review]

**Review by saikumarrs — APPROVED** [review]
